Esmorga-Backend/esmorga-backend
# [MOB-164]  Add user for E2E testing

## Pull Request Checklist

- [x] **[MOB US' Link](https://mobilemakers.atlassian.net/browse/MOB-164)**

## Testing Guide
- `mongo-migrate-ts up` to implement migration 
- Execute a login for example to check user is working 
```
curl --location 'http://localhost:3000/v1/account/login' \
--header 'Content-Type: application/json' \
--data-raw '{
    "email": "auto.esmorga.test@1secmail.com",
    "password": "SuperSecret1!"
}'
```
- `mongo-migrate-ts down -l` to check user is removed
